packages/devtools_app/lib/src/app.dart

```diff
@@ -419,7 +419,7 @@ class OpenAboutAction extends StatelessWidget {
           width: DevToolsScaffold.actionWidgetSize,
           height: DevToolsScaffold.actionWidgetSize,
           alignment: Alignment.center,
-          child: const Icon(
+          child: Icon(
```

Comment:
lots of values have to stop being const due to fewer size and font defaults being const.